api/backups/download_test.go

```diff
@@ -32,7 +33,7 @@ func (s *downloadSuite) TestSuccessfulRequest(c *gc.C) {
 	backupsState := backups.NewBackups(store)
 
 	r := strings.NewReader("<compressed archive data>")
-	meta, err := backups.NewMetadataState(db, "0", "xenial")
+	meta, err := backups.NewMetadataState(db, "0", version.SupportedLts())
```

Comment:
The case was lowered as in https://github.com/juju/utils/blob/master/series/supportedseries.go#L297

I am happy to change it if you really really really want the SupportedLTS. It will make this PR difference surface bigger, of course :D